src/stdlib/SDL_string.c

```diff
@@ -2201,7 +2201,7 @@ int SDL_asprintf(char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, ...)
     return retval;
 }
 
-int SDL_vasprintf(char **strp, const char *fmt, va_list ap)
+int SDL_vasprintf(char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, va_list ap)
```

Comment:
Add `SDL_PRINTF_VARARG_FUNCV(2)` for consistency with above?

Reply:
No, that caused compile errors with clang, and isn't required by Visual Studio.

Reply:
Then remove `SDL_SCANF_VARARG_FUNCV(2)` addition from `SDL_vsscanf` above?

Reply:
BTW, the clang compile errors _should_ go away if you move the position of attribute, like:
```c
int SDL_PRINTF_VARARG_FUNCV(2) SDL_vasprintf (char **strp, SDL_PRINTF_FORMAT_STRING const char *fmt, va_list ap)
```